nautobot/ipam/signals.py

```diff
@@ -0,0 +1,13 @@
+from django.db.models.signals import pre_save
+from django.dispatch import receiver
+
+from nautobot.ipam.models import IPAddress
+
+
+@receiver(pre_save, sender=IPAddress)
```

Comment:
Why does this need to be done as a signal, instead of directly within `IPAddress.save()`?